src/ctap/mod.rs

```diff
@@ -28,7 +28,7 @@ pub mod main_hid;
 mod pin_protocol;
 pub mod response;
 pub mod status_code;
-mod storage;
+pub mod storage;
```

Comment:
I think this should only be `pub` for tests.

```rust
#[cfg(feature = "std")]
pub mod storage;
#[cfg(not(feature = "std"))]
mod storage;
```

Reply:
That would not be necessary if we go through the ctap API to change the storage.